datacats/cli/deploy.py

```diff
@@ -28,8 +27,8 @@ def deploy(environment, opts):
 TARGET_NAME is the name of the environment on DataCats.com. Defaults to
 the environment name.
 """
-    profile = get_working_profile(environment)
-
+    if not profile:
+        raise Exception("Need to pass a loaded user_profile instance")
```

Comment:
Sorry, I misread, that's the part you removed. My comment about the exception stands though